maven-repository-manager/src/main/java/org/jboss/pnc/mavenrepositorymanager/RepositoryManagerDriver.java

```diff
@@ -68,54 +72,65 @@
 
     private final Logger logger = LoggerFactory.getLogger(getClass());
 
-    private Indy indy;
+    private String indyBaseUrl;
+    private Map<String, Indy> indyMap = new HashMap<>();
 
     private List<String> internalRepoPatterns;
 
     @Deprecated
     public RepositoryManagerDriver() { // workaround for CDI constructor parameter injection bug
     }
-
-    @SuppressWarnings("resource")
+    
     @Inject
     public RepositoryManagerDriver(Configuration configuration) {
         MavenRepoDriverModuleConfig config;
-        try {
+		try {
```

Comment:
Stupid eclipse inserted tabs instead of spaces, so the number is correct, but github shows them as 8 spaces while eclipse as 4 :-)